app/misc/repo.ts

```diff
@@ -13,6 +13,79 @@ let span;
 let contributors: [any] = [0];
 let previousOpen;
 let repoName : string = "";
+let jsonfile = require('jsonfile');
+
+// Issue 6
+// Retrieve repos from repos.json
+function getRecentRepositories() {
+    let repoFile = 'repos.json';
+    let repoList;
+
+    try {
+        repoList = JSON.parse(checkFile.readFileSync(repoFile));
+    } catch (err) {
+        console.log('Cannot read ' + repoFile);
+        repoList = {
+            recentRepos: []
+        }
+    }
+    console.log("Collecting recently used repositories: " + repoList.recentRepos);
+    return repoList.recentRepos;
+}
+
+// Issue 6
+// Save repo entry to repos.json
+function saveRecentRepositories(repoPath) {
+    let repoFile = 'repos.json';
+    let repoList;
+    let recentRepos;
+
+    try {
+        repoList = JSON.parse(checkFile.readFileSync(repoFile));
+    } catch (err) {
+        console.log('Cannot read ' + repoFile);
+        repoList = {
+            recentRepos: []
+        }
+    }
+
+    console.log('Updating recent repos');
+    uodatedRepoList = {
```

Comment:
What is this variable? it doesnt seem to be used. Do we just need to call `updateRecentRepos()`

Reply:
Good catch, I forgot to update the `JSON` write.  I wanted to be more explicit about the change such that it was easier to follow the code.  This has been updated, fixed and tested.